umh.docs.umh.app/content/en/docs/getstarted/dataacquisitionmanipulation.md

```diff
@@ -1,205 +1,289 @@
+- [MQTT Simulator](/docs/architecture/microservices/community/mqtt-simulator/)
+- [PackML Simulator](/docs/architecture/microservices/community/packml-simulator/)
+
+## Connect OPC UA Data Sources
+
+OPC UA, often complex, can be streamlined using our Benthos-based OPC UA connector
+accessible from the Management Console.
+
+### Create a Connection with the Management Console
+
+After logging into the Management Console and selecting your instance, navigate
+to the **Data Connections** tab to view existing connections.
+
+![Data Connections Overview](/images/getstarted/dataAcquisitionManipulation/dataConnectionsOverview.png?width=80%)
+
+**Uninitialized Connections** are established but not yet configured as data
+sources, while **Initialized Connections** are fully configured.***h
+
+The health status reflects the UMH-data source connection, not data transmission status.
+
+To add a new connection, click **Add Connection**. Currently, the only option is
+OPC UA Server. Enter the server details, including the unique name and address
+with protocol (`opc.tcp://`) and port (usually `4840`).
+
+For testing with the OPC UA simulator, use:
+
+```text
+opc.tcp://united-manufacturing-hub-opcuasimulator-service:46010
+```
+
+![OPC UA Connection Details](/images/getstarted/dataAcquisitionManipulation/addConnectionDetails.png?width=80%)
+
+Test the connection, and if successful, click **Add Connection**.
+
+### Initialize the Connection
+
+After adding, you must initialize the connection. This creates a new Benthos
+deployment for data publishing to the UMH Kafka broker.
+
+Navigate to **Data Sources** > **Uninitialized Connections** and initiate the
+connection.
+
+![Initialize Connection](/images/getstarted/dataAcquisitionManipulation/initializeConnection.png?width=80%)
+
+Enter authentication details (use _Anonymous_ for no authentication, as with the
+OPC UA simulator).
+
+Specify OPC UA nodes to subscribe to in a yaml file, following the ISA95 standard:
+
+```yaml
+  nodes:
+    - opcuaID: ns=2;s=Pressure,
+      enterprise: pharma-genix,
+      site: aachen,
+      area: packaging,
+      line: packaging_1,
+      workcell: blister,
+      originID: PLC13,
+      tagName: machineState,
+      usecase: _historian
+```
+
+Mandatory fields are `opcuaID`, `enterprise`, `tagName` and `useCase`.
+
+{{% notice note %}}
+Learn more about [Data Modeling in the Unified Namespace](https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/)
+in the Learning Hub.
+{{% /notice %}}
+
+Review and confirm the nodes, then proceed with initialization. Successful
+initialization will be indicated by a green message.
+
+The new data source will now appear in the **Data Sources** section.
+
+![Data Sources Overview](/images/getstarted/dataAcquisitionManipulation/dataSourcesOverview.png?width=80%)
+
+## Connect MQTT Data Sources
+
+MQTT data sources can be connected to UMH exclusively through Node-RED.
```

Comment:
There are MQTT client and server. Client can just connect via the MQTT broker, but servers can be connected either via Node-RED, or by using benthos (i have a config available if you need it). The quickest thing here is to go for Node-RED.